setup.py

```diff
@@ -56,5 +57,15 @@
         'scripts/flytekit_venv'
     ],
     license="apache2",
-    python_requires=">=2.7"
+    python_requires=">=2.7",
+    classifiers=[
+        "Intended Audience :: Developers",
+        "Intended Audience :: System Administrators",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 2.7",
```

Comment:
I think we should stop supporting 2.7 soon actually but we can keep it in for now.

Reply:
yes we should!